lib/src/sqlalchemy/apps/executemany.py

```diff
@@ -0,0 +1,193 @@
+import re
+from typing import Any
+
+import pandas as pd
+import sqlalchemy as sql
+from malevich.square import DF, DFS, Context, processor, scheme
+from pydantic import BaseModel
+from sqlalchemy.exc import SQLAlchemyError
+
+from .models import Query
+
+
+@scheme()
+class ExecuteMessage(BaseModel):
+    command: str
+
+@scheme()
+class FormatTokenMessage(BaseModel):
+    token: str
+    value: str
+
+@scheme()
+class PlaceholderMessage(BaseModel):
+    cmd_id: int
+    token: str
+    value: Any
+
+@processor()
+def executemany(
+    exec_msg: DF[ExecuteMessage],
+    fmt_msg: DF[FormatTokenMessage],
+    plh_msg: DF[PlaceholderMessage],
+    ctx: Context[Query]
+) -> DFS:
+    '''
+    Execute raw SQL commands on the database with many values. Use this only if you need to run same commands with different parameters, since placeholder dataframe is mandatory.
+    If your command does not utilize placeholders, refer to `execute` processor
+
+    ## Input:
+
+        Consists of three dataframes.
+
+        `exec_msg` (DF[ExecuteMessage]): A dataframe with columns:
+            - `command` (str): command string.
+
+        `fmt_msg` (DF[FormatTokenMessage]): A dataframe with format values for tokens in the commands:
+            - `token` (str): token in the command.
+            - `value` (str): substitute string.
+
+        `plh_msg` (DF[PlaceholderMessage]): A dataframe with placeholders for the commands to execute multiple statements:
+            - `cmd_id` (int): id of the command in `exec_msg` dataframe.
+            - `token` (str): name of the token to substitute with.
+            - `value`: value to substitute.
+
+    ## Output:
+
+        A list of dataframes with a result for each query. If the query does not return any values, a dataframe with a single column `rows_affected` is returned.
+
+
+    ## Configuration:
+
+        - `url`: str.
+            URL of the DB to connect to.
+        - `subsequent`: bool.
+            if True, each statement will be commited before the next one is executed.
+
+
+    ## Notes:
+
+        IMPORTANT! Format tokens are needed for literals such as table names and column names. Placeholders are needed for
+        constants and help to execute a command multiple times. In order to use placeholders, mark the tokens in your SQL commands with `:`.
+
+        E.g. for queries:
+        ```sql
+        CREATE TABLE IF NOT EXISTS {table} (
+                {column_1} INTEGER PRIMARY KEY,
+                {column_2} TEXT,
+                {column_3} INTEGER
+            );
+        ```
+        ```sql
+        INSERT INTO {table} ({column_2}, {column_3})
+            VALUES (:name, :price);
+        ```
+
+        The format tokens needed can look like this:
+        ----------------------
+        | token    | value   |
```

Comment:
You are using this table for all the commands. We already partially support configuration override. So the table, that is used for all rows is better to go to the configuration, as it is more light entity than a new collection component. I suggest moving to the configuration (it also consists only two columns, so UI should handle it nicely)